### DISCUSSION TITLE
what's the format of wasb_conn_id in WasbBlobSensor

### DISCUSSION THREAD

**Discussion by ZuoJunLN** [apache/airflow#38685] [Q&A] [answered]
I try to create one Azure Storage account Blob sensor, I would like to know what kind of format for WashBlobSensor:
container_name=<my container name>
blob_name=<my blob name>
wasb_conn_id="" 
my question is what kind of format should I provide in wasb_conn_id, Is it connection string from azure storage account like:
BlobEndpoint=https://<sa name>.blob.core.windows.net/;QueueEndpoint=https://<sa name>.queue.core.windows.net/;FileEndpoint=https://<sa name>.file.core.windows.net/;TableEndpoint=https://<sa name>.table.core.windows.net/;SharedAccessSignature=<sas toke>

**Comment by Taragolis** [comment, marked as answer]
It is a just an identifier and reference to the connection into the:
- Alternate Secrets Backend (optional, if you set it up)
- Environment Variables
- Airflow DB

Some useful links
- [Managing Connections](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html#managing-connections)
- [Secrets Backend](https://airflow.apache.org/docs/apache-airflow/stable/security/secrets/secrets-backend/index.html#secrets-backend)

**Reply by ZuoJunLN** [reply]
Got it thanks!
